CHANGELOG.md

```diff
@@ -184,6 +184,73 @@ your tests.
 
 [#7835]: https://github.com/googleapis/google-cloud-cpp/issues/7835
 
+### New Libraries
+
+We are introducing client libraries for many (>50) GCP services. These libraries
+are still under development.
```

Comment:
I would say: "While we do not anticipate any API changes to these libraries before declaring them GA, we are releasing them early in case they elicit some feedback that requires changes."

Reply:
Fixed and added a bullet for GKE / container